05-descriptive-analysis.Rmd

```diff
@@ -358,7 +360,7 @@ The steps involved are:
 * Specify the cross tab in `group_by()`,
 * Run `survey_mean()` or `survey_prop()` within `summarize()`
 
-We can calculate the weighted average cost of electricity for each region in the U.S.:
+We can calculate the weighted average cost of electricity for each region in the U.S.: 
```

Comment:
Let's first do the overall then do the region.  We do overall for count/total, so let's do it here too to help walk respondents through each possibility.